lib/ain-grpc/src/rpc.rs

```diff
@@ -368,4 +370,9 @@ impl MetachainRPCServer for MetachainRPCModule {
     fn gas_price(&self) -> Result<String> {
         Ok(format!("{:#x}", 0))
     }
+
+    fn get_receipt(&self, hash: H256) -> Result<ReceiptResult> {
+        let receipt = self.handler.receipt.get_receipt(hash).unwrap();
+        Ok(ReceiptResult::from(receipt))
+    }
 }
```

Comment:
```suggestion
    fn get_receipt(&self, hash: H256) -> Result<Option<ReceiptResult>> {
        self.handler
            .receipt
            .get_receipt(hash)
            .map_or(Ok(None), |receipt| Ok(Some(ReceiptResult::from(receipt))))
    }
```
Should return None when not found